content/en/developers/marketplace/api_integration.md

```diff
@@ -0,0 +1,218 @@
+
+## Create an OAuth client
+
+Datadog recommends setting up an [OAuth client][13] to securely authorize third-party application access for your integrations. For more information, see [OAuth for Integrations][14] and [Authorization Endpoints][15].
+
+## Set up a directory and clone the Integrations Extras repository
+
+1. Create a `dd` directory:
+
+   {{< code-block lang="shell" >}}mkdir $HOME/dd{{< /code-block >}}
+   
+   The Datadog Development Toolkit command expects you to be working in the `$HOME/dd/` directory. This is not mandatory, but working in a different directory requires additional configuration steps.
+
+2. Clone the `integrations-extras` repository:
+
+   {{< code-block lang="shell" >}}git clone git@github.com:DataDog/integrations-extras.git{{< /code-block >}}
+
+## Install and configure the Datadog development toolkit
+
+The Datadog Development Toolkit command (`ddev`) allows you to create scaffolding when you are developing an integration by generating a skeleton of your integration tile's assets and metadata.
+
+Before you begin, make sure you meet the following prerequisites:
+
+- [Python v3.8 or later][16]
+- A Python virtual environment is recommended to avoid potential environment conflicts. The instructions below use `venv`, which comes packaged with Python v3.3 and later on most operating systems.
+
+Install and configure the development toolkit:
+
+1. Make sure you're inside the `integrations-extras` directory:
+   {{< code-block lang="shell" >}}cd $HOME/dd/integrations-extras{{< /code-block >}}
+
+2. Set up a Python virtual environment:
+   {{< code-block lang="shell" >}}
+   python3 -m venv venv
+   . venv/bin/activate{{< /code-block >}}
+
+   You can exit the virtual environment at any time by running `deactivate`.
+
+3. Install the [Developer Toolkit][17]:
+   {{< code-block lang="shell" >}}pip3 install "datadog-checks-dev[cli]"{{< /code-block >}}
+
+   If you are using the Z Shell, you may need to use escaped characters by running `pip3 install datadog-checks-dev\[cli\]`.
+
+4. Set `integrations-extras` as the default working repository:
+   {{< code-block lang="shell" >}}
+   ddev config set integrations-extras $HOME/dd/integrations-extras
+   ddev config set repo integrations-extras{{< /code-block >}}
+
+   If you used a directory other than `$HOME/dd` to clone the `integrations-extras` directory, use the following command to set your working repository:
+
+   {{< code-block lang="shell" >}}
+   ddev config set integrations-extras <PATH/TO/INTEGRATIONS-EXTRAS>
+   ddev config set repo integrations-extras{{< /code-block >}}
+
+## Populate the integration tile scaffolding
+
+Run the `ddev` command to generate a skeleton of the folders and files needed for your integration. The options you use with the command are different depending on what type of integration you are developing. For a full list of the files created by the `ddev` command, see [Integrations assets][18].
+
```

Comment:
```suggestion
Run the `ddev` command to generate a skeleton of the folders and files needed for your integration. The options you use with the command are different depending on what type of integration you are developing. For a full list of the files created by the `ddev` command, see [Integrations assets][18]. Since all the code is hosted on the partner's end, only a tile-only listing is required.

```